src/main/java/JobData.java

```diff
@@ -79,14 +79,44 @@ public static ArrayList<HashMap<String, String>> findByColumnAndValue(String col
 
             String aValue = row.get(column);
 
-            if (aValue.contains(value)) {
+            if (aValue.toUpperCase().contains(value)) {
                 jobs.add(row);
             }
         }
 
         return jobs;
     }
 
+    public static ArrayList<HashMap<String, String>> findByValue(String value){
+        loadData();
+        ArrayList <HashMap<String, String>> jobs = new ArrayList<>();
+        for (HashMap<String, String> row: allJobs){
+            for (HashMap.Entry<String, String> job: row.entrySet()){
+                if (job.getValue().toUpperCase().contains(value) && !jobs.contains(row)){
+                    jobs.add(row);
+                }
+            }
+        }
+        return jobs;
+    }
+
+
+
+
+
+    public static ArrayList<HashMap<String, String>> findByValue(String value){
```

Comment:
remove this duplicated method. Having 3 methods that are all named the same in a Class will cause errors

Reply:
Remove the findByValue method below too